### PR TITLE
fix(endorsement-system): Text fixes

### PR DESCRIPTION
**Pull request by alexandradilja**
Some minor text fixes for endorsement system.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/service-portal/endorsements/src/lib/messages.ts [hunk 1]
```diff
@@ -157,6 +157,11 @@ export const m: MessageDir = {
       defaultMessage: 'Ertu viss um að vilja loka lista?',
       description: 'Dialog prompt title',
     },
+    dialogPromptLockListTitle: {
+      id: 'sp.petitions:dialog-prompt-close',
```
Comment (albinagu) [inline libs/service-portal/endorsements/src/lib/messages.ts]: prompt-lock

File: libs/service-portal/endorsements/src/lib/messages.ts [hunk 2]
```diff
@@ -172,6 +177,11 @@ export const m: MessageDir = {
       defaultMessage: 'Loka lista',
       description: 'Button label',
     },
+    lockListButton: {
+      id: 'sp.petitions:close-list',
```
Comment (albinagu) [inline libs/service-portal/endorsements/src/lib/messages.ts]: lock-list

það er til close-list, lína 176

Comment (albinagu) [reply libs/service-portal/endorsements/src/lib/messages.ts]: ef við erum ekki að nota closeListButton lengur, þá kannski eyða þeim message bara

File: libs/service-portal/endorsements/src/lib/messages.ts [hunk 3]
```diff
@@ -207,6 +217,11 @@ export const m: MessageDir = {
       defaultMessage: 'Ekki tókst að loka lista',
       description: 'Toast error message',
     },
+    toastErrorLockList: {
+      id: 'sp.petitions:toast-error-message-close',
```
Comment (albinagu) [inline libs/service-portal/endorsements/src/lib/messages.ts]: lock

File: libs/service-portal/endorsements/src/lib/messages.ts [hunk 4]
```diff
@@ -222,6 +237,11 @@ export const m: MessageDir = {
       defaultMessage: 'Tókst að loka lista',
       description: 'Toast success message',
     },
+    toastSuccessLockList: {
+      id: 'sp.petitions:toast-success-message-close-list',
```
Comment (albinagu) [inline libs/service-portal/endorsements/src/lib/messages.ts]: lock-list

**Review by albinagu — APPROVED** [review]

File: libs/service-portal/endorsements/src/lib/messages.ts (hunk 2 above)
Comment (alexandradilja) [reply libs/service-portal/endorsements/src/lib/messages.ts]: verið að nota í veitPetition fyrir eigendur

**Pull request merged**